OJP/OJP_Common.xsd

```diff
@@ -449,7 +449,21 @@
 					<xs:documentation>Internal code of the attribute. Can be used for detection of double occurrences.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:group ref="AllFacilitiesGroup"/>
```

Comment:
I think the AllFacilitiesGroup isn't used anywhere any more, could be removed...?

Reply:
removed